packages/contracts-bedrock/interfaces/L1/IOPContractsManager.sol

```diff
@@ -209,6 +210,8 @@ interface IOPContractsManager {
 
     error SuperchainProxyAdminMismatch();
 
+    error PrestateNotSet();
```

Comment:
`SuperchainProxyAdminMismatch` and `PrestateNotSet` are missing natspec

Reply:
actually so are a bunch of other errors in this file further above

Reply:
I will open another PR to fix that.